packages/sw-broadcast-cache-update/src/lib/behavior.js

```diff
@@ -64,27 +65,22 @@ import {defaultHeadersToCheck, defaultSource} from './constants';
  *     cacheName
  *   });
  * }
+ *
+ * @memberof module:sw-broadcast-cache-update
  */
 class Behavior {
   /**
-   * Creates a new `Behavior` instance, which is used to compare two
-   * [Responses](https://developer.mozilla.org/en-US/docs/Web/API/Response)
-   * and use the {@link https://developers.google.com/web/updates/2016/09/broadcastchannel|Broadcast Channel API}
-   * to notify interested parties when those Responses differ.
-   *
-   * For efficiency's sake, the underlying response bodies are not compared;
-   * only specific response headers are checked.
+   * To instantiate this Class you must provide a channel name which
+   * will be used to when creating a broadcast channel to dispatch messages on.
```

Comment:
I've reworded this sentence - still not great but I'm struggling with what to put.